packages/spec-driver-selenium/src/spec-driver.ts

```diff
@@ -63,6 +63,16 @@ export function transformSelector(selector: Selector | CommonSelector): Selector
   }
   return selector
 }
+export function untransformSelector(selector: Selector): CommonSelector {
+  if (utils.types.isString(selector)) {
+    return {type: 'css', selector: selector}
+  } else if (byHash.includes(Object.keys(selector)[0])) {
+    const s = Object.entries(selector)[0]
+    return {type: s[0], selector: s[1]}
```

Comment:
The problem with that is the fact that `By.linkText('...')` and `{linkText: '...'}` will generate different common selectors.
I would better do something like this
```suggestion
    const [[method, selector]] = Object.entries(selector)
    const by = Selenium.By[method](selector)
    return {type: by.using, selector: by.value}
```